src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -147,3 +147,44 @@ def test_containerapp_identity_user(self, resource_group):
         self.cmd('containerapp identity show -g {} -n {}'.format(resource_group, ca_name), checks=[
             JMESPathCheck('type', 'None'),
         ])
+
+
+@live_only()
```

Comment:
May I ask why this test is live only?

Reply:
@haroonf is this because of the LogAnayltics ? If so, please update this as others as well.